flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaTableTestBase.java

```diff
@@ -95,4 +142,71 @@ public void deleteTestTopic(String topic) {
             admin.deleteTopics(Collections.singletonList(topic));
         }
     }
+
+    // ------------------------ For Debug Logging Purpose ----------------------------------
+
+    protected void scheduleTimeoutLogger(Duration period, Runnable loggingAction) {
```

Comment:
These methods can be `private`.
- scheduleTimeoutLogger
- cancelTimeoutLogger
- describeExternalTopics
- logTopicPartitionStatus